### ISSUE TITLE
BATCH_NORM

### ISSUE THREAD

**Issue by liu666666** [facebook/C3D#280]
@dutran  Thanks for your nice work!
I have two questions:
1.The value of batch size in train.prototxt is 24, I use 4 gpus with nccl, and the value of iter_size in solver.prototxt is 2. So my real batch_size is 24*4*2=192.Right?
2.The function of BatchNorm layer is to normalise the input to have 0-mean and/or unit (1) variance across the batch.  what is the batch in your code If I have set the parameters as mentioned in (1)? 24 or 96 or 192?

**Comment by dutran** [comment]
1. 24 * 4 = 94, yes. (which solver.prototxt file are you referring to). I think your real batch size is 96 and you should adjust stepsize accordingly 
2. On training batch norm works locally, not across GPUs. So it will normalize based on the local statistics of 24 examples, not 96.
